help/print-a-conversation.md

```diff
@@ -0,0 +1,45 @@
+# Print a conversation
+
+Conversations, including topics, DMs, and group DMs, are printable from the
+Zulip web app.
+
+{start_tabs}
+
+{tab|web}
+
+1. Navigate to the topic, DM, or group DM you wish to print.
+
+1. Ensure that all messages you wish to print from further back in time
+   have been loaded and can be scrolled to on screen (all loaded messages
+   will print).
+
+1. Use your browser's print feature (e.g., File > Print) to preview
+   and print your messages.
+
+{tab|mobile}
+
+1. Navigate to the topic, DM, or group DM you wish to print.
+
+1. Ensure that all messages you wish to print from further back in time
+   have been loaded and can be scrolled to on screen (all loaded messages
+   will print).
+
+1. Use your mobile browser's print feature to preview and print your messages.
```

Comment:
Generally, we only document mobile features specific to the Zulip mobile app. I think we can remove this section since the instructions for `web` would also apply when using a mobile browser.